apps/test-bundles/webpack.config.js

```diff
@@ -0,0 +1,86 @@
+const path = require('path');
+const resources = require('../../scripts/tasks/webpack-resources');
+
+const AllEntries = [
```

Comment:
I am not sure how to get this list in the best way.

We could put an "entries" section in the `office-ui-fabric-react/package.json` file and read that.

Reply:
Dynamically generate by reading the files at the top level of OUFR that looks like a top level import? (i.e. all pascal cased files?)